src/App.css

```diff
@@ -0,0 +1,67 @@
+.App{
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+    justify-content: stretch;
+}
+/* Tags */
+
+.tags-container{
+    display: flex;
+    gap: 5px;
+    flex-wrap: wrap;
+}
+
+.tag{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 5px;
+    background-color: var(--COLOR-SHADOW-PRIMARY-400);
+    border-radius: 99rem;
+    padding: 3px 5px ;
+}
+
+.tag-title{
+    font-size: 1.4rem;
+    font-weight: 500;
+    color: var(--OFF-BLACK);
+    padding: 0 3px;
+}
+
+.tag-close{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    padding: 2px;
+    border:0;
+    border-radius: 999px;
```

Comment:
Why are we using ```border-radius:999px``` here?

Reply:
To make the button round. I generally use 99px  so that even if button size shrink or grows it should not affect the radius that I wish to keep round so even if button enlarge still it remains round

Reply:
you can just use border radius of 50% for making a button round